From cab670bab2e60d995dbedb2e8fbf6ddc9775e270 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Fri, 1 Aug 2025 13:24:22 +0000
Subject: [PATCH] Always present window on launch

This ensures that the main window is brought to the front even if it's
already launched.
---
 lollypop/application_cmdline.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/lollypop/application_cmdline.py b/lollypop/application_cmdline.py
index 89572d080..0799dab33 100644
--- a/lollypop/application_cmdline.py
+++ b/lollypop/application_cmdline.py
@@ -199,10 +199,9 @@ class ApplicationCmdline:
                 else:
                     self.__on_parse_finished(None, None, [], audio_uris)
             elif self.window is not None:
-                if not self.window.is_visible():
-                    self.window.present()
-                    emit_signal(self.player, "status-changed")
-                    emit_signal(self.player, "current-changed")
+                self.window.present()
+                emit_signal(self.player, "status-changed")
+                emit_signal(self.player, "current-changed")
             Gdk.notify_startup_complete()
         except Exception as e:
             Logger.error("Application::__on_command_line(): %s", e)
-- 
GitLab

